models/Hotspot.ts

```diff
@@ -2,9 +2,16 @@ import { camelCase } from 'lodash';
 import doneMarker from '../assets/marker-done.png';
 import inProgressMarker from '../assets/marker-in-progress.png';
 import todoMarker from '../assets/marker-todo.png';
+import { isWeb } from '../utils/platform';
 import { castToCat, Cat } from './Cat';
 import { castToUser, User } from './User';
 
+const doneMarkerImage = isWeb() ? '/assets/assets/marker-done.png' : doneMarker;
+const inProgressMarkerImage = isWeb()
+  ? '/assets/assets/marker-in-progress.png'
+  : inProgressMarker;
+const todoMarkerImage = isWeb() ? '/assets/assets/marker-todo.png' : todoMarker;
```

Comment:
this can be an object to avoid the duplication of `isWeb()` check: 

```
const markers = isWeb() : {
  doneMarkerImage: '/assets/assets/marker-done.png' ,
  inProgressMarkerImage: '/assets/assets/marker-in-progress.png' ,
  todoMarkerImage: '/assets/assets/marker-todo.png' ,
} :{
  doneMarkerImage: doneMarker,
  inProgressMarkerImage: inProgressMarker,
  todoMarkerImage: todoMarker,
}
```